src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -635,3 +635,21 @@ def _configure_security_policy_storage_params(cmd):
                options_list=('--end-ip-address',),
                help='The end IP address of the firewall rule. Must be IPv4 format. Use value'
                ' \'0.0.0.0\' to represent all Azure-internal IP addresses.')
+
+
+#####
+#           sql server vnet-rule
+#####
+
+
+with ParametersContext(command='sql server vnet-rule') as c:
+    # Help text needs to be specified because 'sql server firewall-rule update' is a custom
+    # command.
+    c.argument('server_name',
+               options_list=('--server', '-s'))
```

Comment:
There should be a completer for the server name.

Reply:
Here's an example for completer:
https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py#L193

You can use that method to get the list of server names.

Reply:
Not sure what a completer is - can we use the server_param_type like in line 506?

 c.argument('server_name', arg_type=server_param_type)

Reply:
Completer is a keyword argument.  You could use `server_param_type` but the completer would still need to be part of that type. :)  The above line shows how you would enable the completer for this situation.  It should be very easy.